docs/examples/good_practices/launch_many_jobs/README.rst

```diff
@@ -0,0 +1,292 @@
+.. NOTE: This file is auto-generated from examples/good_practices/launch_many_jobs/index.rst
+.. This is done so this file can be easily viewed from the GitHub UI.
+.. **DO NOT EDIT**
+
+.. _launch_many_jobs:
+
+Launch many jobs from same shell script
+=======================================
+
+Sometimes you may want to run same script by changing just few values.
+
+Instead of "comment line, change value, and re-run" cycle, you can
+parameterize the script and then call it multiple times with different parameters.
+
+Here we provide an example, with Python script accepting parameters, and
+bash script just receiving and passing parameters to Python script.
```

Comment:
```suggestion
Sometimes you may want to run the same job with different arguments. For example, you may want to launch an experiment using a few different values for a given parameter. 

The naive way to do this would be to create multiple sbatch scripts, each with a different value for that parameter.
Another might be to use a single sbatch script with multiple lines, each with a different parameter value, and to then uncomment a given line before submitting the job, then commenting and uncommenting a different line before submitting another job, etc. 

This example shows a  practical solution to this problem, allowing you to parameterize an job's sbatch script, allowing you to pass different values directly from the command-line when submitting the job.

In this example, our job script is a slightly modified version of the Python script form the single-GPU example, with a bit of code added so that it is able to take in values form the command-line.
The sbatch script uses the `$@` bash directive to pass the command-line arguments to the python script. This makes it very easy to submit multiple jobs, each with different values!

The next examples will then build on top of this one to illustrate good practices related to launching lots of jobs for hyper-parameter sweeps:
* Using SLURM Job Arrays for Hyper-Parameter Sweeps (coming soon!)
* :doc:`Running more effective Hyper-Parameter Sweeps with Orion </examples/good_practices/hpo_with_orion/index>` 

```